content/docs/run-api-server/configuring.mdx

```diff
@@ -48,10 +48,13 @@ As outlined at the beginning, we presume you are interested in starting an inges
 | `--ingest` | `INGEST` | true |
 | `--captive-core-config-path` | `CAPTIVE_CORE_CONFIG_PATH` | /etc/default/stellar-captive-core.toml |
 | `--stellar-core-binary-path` | `STELLAR_CORE_BINARY_PATH` | /usr/bin/stellar-core |
+| `--captive-core-use-db` | `CAPTIVE_CORE_USE_DB` | true |
 
   - The first parameter, `--ingest=true`, enables the ingestion subservice.
   - The second parameter, `--captive-core-config-path`, points to a Captive Core configuration file. This TOML file only requires a few fields (explained [below](#configuring-captive-core)) to get up and running.
-  - The final parameter, `--stellar-core-binary-path`, is a filesystem path to a Stellar Core binary. Horizon will actually search your PATH for `stellar-core` by default, so if your environment is configured appropriately, you won't even need to pass this.
+  - The third parameter, `--stellar-core-binary-path`, is a filesystem path to a Stellar Core binary. Horizon will actually search your PATH for `stellar-core` by default, so if your environment is configured appropriately, you won't even need to pass this.
+  - The fourth parameter, `--captive-core-use-db=true`, enable this captive core based ingestion to use a local file DB rather than in-memory(RAM) for ledger states. Essentially moves what would have been stored in RAM to the local file DB instead. Recent profiling on the two approaches shows an approximate space usage of about 8GB for ledger states as of 02/2022 timeframe, but will continue to increase as network ledger grows over time. The location of local file DB is determined by the `DATABASE` parameter in the `--captive-core-config-path` file. If no value is set, by default it will be `sqlite3://stellar.db` which resolves to runtime directory location derived from `--captive-core-storage-path` or current runtime working directory.
```

Comment:
Grammatical, clarity, and contextual suggestions:

```suggestion
  - The fourth parameter, `--captive-core-use-db=true`, enables Captive Core ingestion to use store ledger states a local SQLite database rather than in memory (RAM). As of this writing, ledger states require approximately 8GB, but this will continue to increase as the ledger grows over time. The database location is determined by the `DATABASE` parameter within the `--captive-core-config-path` file. By default, it is set to  `sqlite3://stellar.db`, which resolves to runtime directory location derived from `--captive-core-storage-path`.
```

Is it accurate that it has to be a sqlite path?

Reply:
yes, part of the changeset included validation on the DATABASE to be sqlite3 path.
https://github.com/stellar/go/pull/4092/files#diff-c7702ef819e52917ed3868588dff5273774c61a6aa8d36c1644d7ba251fb7756R561

came about after a slack chat with core team on DATABASE, mentioned it has primarily been a sqlite3 connection, it potentially could take other db urls like postgres, but not widely used yet, so for this initial captive core db usage, was determined to limit setting to most stable.

Reply:
Included recommended grammar change - [8e4ae8afc39](https://github.com/stellar/new-docs/pull/644/commits/8e4ae8afc3964ef65737b61020eb2bfe9e95c09f)